nf_core/pipeline-template/.github/workflows/ci.yml

```
@@ -18,21 +18,28 @@ jobs:
       NXF_ANSI_LOG: false
     strategy:
       matrix:
-        # Nextflow versions: check pipeline minimum and latest edge version
-        nxf_ver:
-         - 'NXF_VER=21.10.3'
-        #  - 'NXF_EDGE=1'
+        # Nextflow versions
+        include:
+          # Test pipeline minimum Nextflow version
+          - NXF_VER: '21.10.3'
+            NXF_EDGE: ''
+          # Test latest edge release of Nextflow
+          - NXF_VER: ''
+            NXF_EDGE: '1'
     steps:
       - name: Check out pipeline code
         uses: actions/checkout@v2
 
       - name: Install Nextflow
         env:
           CAPSULE_LOG: none
+          {% raw -%}
+          NXF_VER: ${{ matrix.NXF_VER }}
+          NXF_EDGE: ${{ matrix.NXF_EDGE }}
```

x-ref https://github.com/nextflow-io/nextflow/issues/2467
```suggestion
          # NXF_EDGE: ${{ matrix.NXF_EDGE }}
```